app/src/main/java/com/saucelabs/mydemoapp/android/view/activities/CustomFeedbackFormActivity.java

```diff
@@ -8,9 +8,7 @@
 
 import com.saucelabs.mydemoapp.android.R;
 import com.saucelabs.mydemoapp.android.databinding.ActivityCustomFeedbackFormBinding;
-import com.saucelabs.mydemoapp.android.utils.TestFairyAssetReader;
 import com.saucelabs.mydemoapp.android.utils.base.BaseActivity;
-import com.testfairy.TestFairy;
 
 
 public class CustomFeedbackFormActivity extends BaseActivity {
```

Comment:
I think this entire file can go away since all of it was about TestFairy SDK.

Reply:
yup! we can remove this.